packages/widget/src/types/events.ts

```diff
@@ -19,6 +19,7 @@ export enum WidgetEvent {
   WalletConnected = 'walletConnected',
   WidgetExpanded = 'widgetExpanded',
   PageEntered = 'pageEntered',
+  SendToWalletAddressChanged = 'sendToWalletAddressChanged',
```

Comment:
If we add all fields, we should come up with one event that has a field name and value. Wdyt?

Maybe something like 

- FormFieldChanged
- FormValueChanged

Reply:
Makes sense, when doing this I did wonder it we would end you wanting to do it for all values

Reply:
have gone with this approach using FormFieldChanged, I'm gonna look at better type inference as well. Also I think I used value rather than newValue so will change that

Reply:
I've improved the types now as well